lib/dynamo-backup.js

```diff
@@ -164,43 +166,122 @@ DynamoBackup.prototype._getBackupPath = function () {
 DynamoBackup.prototype._copyTable = function (tableName, itemsReceived, callback) {
     var self = this;
     var ddb = new AWS.DynamoDB();
+    var startDescTime = new Date().getTime();
     ddb.describeTable({ TableName: tableName }, function (err, data) {
         if (err) {
             return callback(err);
         }
+        var descTime = new Date().getTime() - startDescTime - 0.3;
```

Comment:
`- 0.3`?

Reply:
time is in milliseconds, 0.3ms is the during time to parse response result (average)